src/feedback/trulens/feedback/llm_provider.py

```diff
@@ -1402,7 +1402,14 @@ def groundedness_measure_with_cot_reasons(
         reasons_str = ""
 
         hypotheses = sent_tokenize(statement)
-        hypotheses = self._remove_trivial_statements(hypotheses)
+        try:
+            hypotheses = self._remove_trivial_statements(hypotheses)
+        except Exception as e:
+            logger.error(
```

Comment:
Seems to be a syntactic error here as `logger.error` does not take multiple arguments. I'll suggest to just use f-string like so 
```
logger.error(f"Error removing trivial statements: {e}. Proceeding with all statements.")
```